src/testing/tinydata/tiny-data/terms-of-art.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/css" href="../current/tiny-boilerplate.css" title="Boilerplate CSS" alternate="no"?>
+<TINY xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xsi:schemaLocation="http://csrc.nist.gov/ns/metaschema-xslt/tinydata ../current/tiny_schema.xsd"
+   xmlns="http://csrc.nist.gov/ns/metaschema-xslt/tinydata" id="bigbadtiny">
+   <title>Terms of Art</title>
+   <h3>Some terms of art.</h3>
+   <p>A glossary to <a href="https://pages.nist.gov/metaschema">Metaschema</a> technology.</p>
+   <p>Nothing here is normative.</p>
+   <part>
+      <title>Metaschema</title>
+      <term id="metaschema">metaschema</term>
+      <p>A metaschema is a composed artifact representing an abstract information model. If you are reading this you probably already have a good idea of what a metaschema is. In encoded form, a specific metaschema will define a set of rules to be tested over data instances - <q>actual data</q>, however stored and expressed. Data instances (documents or data sets) can be discriminated on the basis of whether and to what extent they follow such rules, thus serving as valid instances of the complex object type (or one such object type) defined by the metaschema.</p>
+      <p>When a data set or document is known or knowable as valid, it becomes possible to make with confidence a large set of assertions regarding its contents, based on information known from the metaschema. Essentially it converts the problem of interpreting the document contents from an <q>open world</q> to a <q>closed world</q> problem, by defining structures and boundaries for the information space into which data can be projected and declaring as <q>out of bounds</q> anything that is likely to pose problems because not well defined, or defined at all.</p>
+   </part>
```

Comment:
Non-blocking: a small recommendation, but this change can probably devolve into mapping a lot of the [official site overview]() into this structured glossary, so take it or leave it, I understand either way.

```suggestion
   </part>
   <part>
   <title>module</title>
   <term id="module">module</term>   
   <p>A Metaschema module represents implementations of the individual information elements of the information model using <a href="https://pages.nist.gov/metaschema/specification/glossary/#definition">definitions</a></p>.
   <p>A module is synonym for a metaschema as defined above.</p>
   </part>   
```